in_toto/runlib.py

```diff
@@ -818,11 +842,21 @@ def in_toto_record_stop(step_name, product_list, signing_key=None,
     unfinished_fn = UNFINISHED_FILENAME_FORMAT.format(step_name=step_name,
         keyid=signing_key["keyid"])
 
+    if metadata_directory is not None:
+      unfinished_fn = os.path.join(metadata_directory, unfinished_fn)
+
   # FIXME: Currently there is no way to know the default GPG key's keyid and
   # so we glob for preliminary link files
+  # FIXME: UNFINISHED_FILENAME_FORMAT_GLOB_DIR has a problem with window system
   else:
     unfinished_fn_glob = UNFINISHED_FILENAME_FORMAT_GLOB.format(
         step_name=step_name, pattern="*")
+
+    if metadata_directory is not None:
+      unfinished_fn_glob = UNFINISHED_FILENAME_FORMAT_GLOB_DIR.format(
```

Comment:
I think this needs to be imported. https://travis-ci.org/github/in-toto/in-toto/jobs/693631357#L259